gtest/src/manager.rs

```diff
@@ -24,11 +24,13 @@ use core_processor::{common::*, configs::BlockInfo, Ext};
 use gear_backend_wasmtime::WasmtimeEnvironment;
 use gear_core::{
     memory::PageNumber,
-    message::{Dispatch, DispatchKind, Message, MessageId},
+    message::{Dispatch, DispatchKind, Message, MessageId, Payload},
     program::{CodeHash, Program as CoreProgram, ProgramId},
 };
 use std::collections::{BTreeMap, VecDeque};
 use std::time::{SystemTime, UNIX_EPOCH};
+use std::collections::HashMap;
+use std::fmt::Debug;
```

Comment:
We may nest `std` too.

https://doc.rust-lang.org/stable/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#using-nested-paths-to-clean-up-large-use-lists